common/lib/src/realm_common_base.dart

```diff
@@ -57,10 +57,12 @@ class RealmModel {
   const RealmModel([this.type = ObjectType.realmObject]);
 }
 
-/// MapTo annotation for class member level.
+/// MapTo annotation for class level and class member level.
 ///
-/// Indicates that a property should be persisted under a different name.
-/// This is useful when opening a Realm across different bindings where code style conventions might differ.
+/// MapTo on class level indicates that a class should be persisted under a different collection name.
+/// MapTo on class member level indicates that a property should be persisted under a different name.
```

Comment:
The text now is too verbose imo. I would suggest keeping the previous text.
So 
```
/// Indicates that the class or the property should be persisted under a different name

```

Reply:
Ok. Done.